GetStream/stream-chat-android
# Fix updating the DB after sending the message

### 🎯 Goal

Fix updating the DB after sending the message

### 🛠 Implementation details

We didn't properly update the DB after receiving the result from `sendMessage` call.
We should put `processedMessage`, which is a response from the API because it might be enriched with different info (e.g. commands) and might have different `type`, if the message from the DB wasn't synced yet.

### 🧪 Testing

1. Try to send a Giphy and observe it works fine
2. Try to do the same on `develop` and observe the difference

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
